### PR TITLE
[sd-3986]-(fix:[SD-3994]) - Keyboard operation and focus issues in su…

### PR DESCRIPTION
**Pull request by sjunaid**
…bject metadata drop-down in search filter panel

### CODE REVIEW

File: scripts/superdesk-search/views/item-search.html [hunk 1]
```diff
@@ -19,23 +19,38 @@
...
             <div class="control">
-                <input type="text" id="search-headline" ng-model="meta.headline">
+                <input type="text" id="search-slugline" ng-model="meta.slugline">
             </div>
         </div>
-
         <div class="field">
-            <label for="search-slugline">
-                {{:: 'Slugline' | translate}}
+            <label for="search-subject">
```
Comment (petrjasek) [inline scripts/superdesk-search/views/item-search.html]: just wondering why moving those fields around?

Comment (sjunaid) [inline scripts/superdesk-search/views/item-search.html]: just arranged in similar order that appears in header section of authoring, slugline > subject > headline and so on.

Comment (mdhaman) [inline scripts/superdesk-search/views/item-search.html]: would be worth confirming the order with Andrew.

Comment (sjunaid) [inline scripts/superdesk-search/views/item-search.html]: just confirmed, no issues.

**Comment by mdhaman** [comment]
:+1:

**Comment by petrjasek** [comment]
:+1:

**Pull request merged**